lib/std/fs.zig

```diff
@@ -621,6 +628,7 @@ pub const Dir = struct {
         InvalidUtf8,
         BadPathName,
         DeviceBusy,
+        NotCapable,
```

Comment:
Same question here and everywhere else we find AccessDenied and NotCapable in the same error set.